scripts/export-test-result.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const { getSecretKey, executeCommand, printLog, logAndExit } = require('./utils')
+
+async function main() {
+  await exportTestResult('unit', ['test-report/unit/', 'test-report/unit-bs/'])
+  await exportTestResult('e2e', ['test-report/e2e/', 'test-report/e2e-bs/'])
+}
+
+async function exportTestResult(type, folders) {
+  const ddCiApiKey = (await getSecretKey('ci.browser-sdk.datadog_ci_api_key')).trim()
+
+  await executeCommand(
+    `npx @datadog/datadog-ci junit upload --service browser-sdk --env ci --tags type:${type} ${folders.join(' ')}`,
```

Comment:
It's a bit sad that it can't be used as a library. Anyway, I'd use `yarn` instead of `npx` since it is our package manager of choice :)

```suggestion
    `yarn datadog-ci junit upload --service browser-sdk --env ci --tags type:${type} ${folders.join(' ')}`,
```